osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -462,14 +475,18 @@ private void pushWhenLoaded()
 
                 // only show if the warning was created (i.e. the beatmap needs it)
                 // and this is not a restart of the map (the warning expires after first load).
+                if (epilepsyWarning?.IsAlive == true || lovedWarning?.IsAlive == true)
+                {
+                    pushSequence.TransformBindableTo(volumeAdjustment, 0.25, EpilepsyWarning.FADE_DURATION, Easing.OutQuint);
```

Comment:
I think this is missing the `.Delay(CONTENT_OUT_DURATION)` from the original call site. Means the volume fade it going to start sooner than it used to.